README.md

```diff
@@ -1,48 +1,49 @@
 [![Build Status](https://travis-ci.com/vcoutasso/Chessqdl.svg?branch=develop)](https://travis-ci.com/vcoutasso/Chessqdl)
 
-
 # ChessQDL
 
+<h1 align="center">
+    <img alt="Screenshot ChessQDL" src="https://vcoutasso.com/assets/images/chess.png" />
+</h1>
```

Comment:
A paragraph tag is probably more appropriate than a h1 here (gets rid of the extra line)